docs/manifest.json

```diff
@@ -637,8 +637,7 @@
           ["SignUpInitialValues", "/references/javascript/types/sign-up-initial-values"],
           ["MagicLinkError", "/references/javascript/types/magic-link-error"],
           ["DeletedObject", "/references/javascript/types/deleted-object"],
-          ["OAuthProvider", "/references/javascript/types/oauth-provider"],
-          ["OAuthStrategy", "/references/javascript/types/oauth-strategy"],
+          ["OAuthProvider and OAuthStrategy", "/references/javascript/types/oauth"],
```

Comment:
I think combining these is probably a good idea, but I wanted to note that the page name looks crowded in the sidebar in the preview. I don't have a suggestion for a fix, and I don't think it should block this PR, but wanted to point it out
![Uploading image.png…]()

Reply:
I honestly wouldn't mind naming the page OAuth types since that's its title in the actual markdown and SEO. Of course it's different from how every other page in this section is named but I don't think it's different in a confusing way. I leave it up to you though as I don't think it's a huge deal